source/extensions/all_extensions.bzl

```diff
@@ -50,14 +50,8 @@ def envoy_all_http_filters():
 
     return {_selected_extension_target(v): True for k, v in all_extensions.items() if (k.startswith(_http_filter_prefix) or k.startswith(_upstream_http_filter_prefix))}.keys()
 
-# All network-layer filters are extensions with names that have the following prefix.
-_network_filter_prefix = "envoy.filters.network"
-
-# All thrift filters are extensions with names that have the following prefix.
-_thrift_filter_prefix = "envoy.filters.thrift"
-
 # Return all network-layer filter extensions to be compiled into network-layer filter generic fuzzer.
-def envoy_all_network_filters():
+def envoy_select_network_filters(filters):
```

Comment:
The function no longer deals with network filters, so maybe rename to `envoy_select_filters_from_name`

Reply:
Mhh, I propose `envoy_network_filters_from_selected`. How do you like that?